tools/image_processing/bia-ftplinks/biaftplink.xml

```diff
@@ -0,0 +1,41 @@
+<tool id="bia_ftplink" name="FTP Link for Bioimage Archive" version="@VERSION@+galaxy1">
+    <description>Get FTP link for a study from Bioimage Archive</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <command detect_errors="aggressive">
+	    <![CDATA[
+		#for $study in $study_name.split(","):
```

Comment:
I guess this tool is a good step forward, but wouldn't it be better to query the BIA API?

Reply:
i thought the curl command actually queries the api..I haven't found other api documentation yet..

Reply:
The BIA API seems to be the BioStudies one: https://www.ebi.ac.uk/biostudies/help

Reply:
Yes, its an API. Sorry I was not concrete here. I think its a good first step. I was just wondering if there is no API abstraction library that we should use. But maybe there is none.

Reply:
thanks both. Yes I found that biostudies API. It seems more complicated than just a tool for getting FTP links.